src-plugins/manualRegistration/manualRegistrationToolBox.cpp

```diff
@@ -499,9 +526,13 @@ void manualRegistrationToolBox::displayButtons(bool show)
     }
 }
 
-
 void manualRegistrationToolBox::updateLabels(int left,int right)
```

Comment:
(done, even if it does not appear on the Conversation tab of this PR. It's strange, on the File Changed tab, it's seen)